app/presenters/world_wide_taxon_presenter.rb

```diff
@@ -0,0 +1,109 @@
+class WorldWideTaxonPresenter
+  ACCORDION = :accordion
+  LEAF = :leaf
+
+  attr_reader :taxon
+  delegate(
+    :content_id,
+    :title,
+    :description,
+    :base_path,
+    :tagged_content,
+    :child_taxons,
+    :live_taxon?,
+    :most_popular_content,
+    :can_subscribe?,
+    to: :taxon
+  )
+
+  def initialize(taxon)
+    @taxon = taxon
+  end
+
+  def rendering_type
+    taxon.children? ? ACCORDION : LEAF
+  end
+
+  def renders_as_accordion?
+    rendering_type == ACCORDION
+  end
+
+  GENERAL_INFORMATION_TITLE = 'General information and guidance'.freeze
+
+  def accordion_content
+    return [] unless renders_as_accordion?
+    accordion_items = ordered_child_taxons.map { |taxon| WorldWideTaxonPresenter.new(taxon) }
+
+    if tagged_content.any?
+      general_information = WorldWideTaxon.new(
+        ContentItem.new(
+          'content_id' => taxon.content_id,
+          'title' => GENERAL_INFORMATION_TITLE,
+          'base_path' => GENERAL_INFORMATION_TITLE.downcase.tr(' ', '-'),
+          'description' => ''
+        )
+      )
+      general_information.can_subscribe = false
+
+      accordion_items.unshift(
+        WorldWideTaxonPresenter.new(general_information)
+      )
+    end
+
+    accordion_items
+  end
+
+  def options_for_leaf_content(index:)
+    {
+      module: 'track-click',
+      track_category: 'navLeafLinkClicked',
+      track_action: (index + 1).to_s,
+      track_label: tagged_content[index].base_path,
+      track_options: { dimension28: tagged_content.size.to_s,
+                       dimension29: tagged_content[index].title }
+    }
+  end
+
+  def options_for_child_taxon(index:)
+    {
+      module: 'track-click',
+      track_category: 'navGridContentClicked',
+      track_action: (index + 1).to_s,
+      track_label: child_taxons[index].base_path,
+      track_options: { dimension26: tagged_content.any? ? '2' : '1',
```

Comment:
I'd be tempted to _golf down_ these tracking dimension hashes, there's a lot of similarity in the following 2 methods.

Reply:
@steventux It's only the `module` that's consistent across all of the methods. They are just different enough that I'd keep them separate for clarity.

Reply:
Ok, I'll kill the commit.